### DISCUSSION TITLE
❓ VAD Training Data Used

### DISCUSSION THREAD

**Discussion by stonelazy** [snakers4/silero-vad#99] [Q&A]
## ❓ VAD Training Data

Am not sure if this is morally right to ask this question, forgive me if am wrong. Could you let me know what were the data that was used to train silero_vad ? Whether it is a proprietary data or any public dataset ? If latter, could you pl name it ?  
Are there any recommended datasets that you would suggest to train VAD with ?

**Comment by snakers4** [comment]
Answered here - https://github.com/snakers4/silero-vad/discussions/78

Adapting VAD to new domains / improving it is one of commercial services we provide
